automated install/uninstall.sh

```diff
@@ -41,10 +36,13 @@ else
     fi
 fi
 
-readonly PI_HOLE_FILES_DIR="/etc/.pihole"
+# Other PI_HOLE_* vars sourced from basic-install.sh
+PI_HOLE_GIT_DIR="/etc/.pihole"
```

Comment:
I'm wondering if it might be better to exclude uninstall.sh from this PR?

Instead, I would alter #6339 to use the new variable names from this PR

(PI_HOLE_GIT_DIR in place of PI_HOLE_LOCAL_REPO, PI_HOLE_SCRIPT_DIR in place of PI_HOLE_INSTALL_DIR)

Reply:
although, that depends upon which order they get merged I suppose.